### ISSUE TITLE
Better Config File/Instructions

### ISSUE THREAD

**Issue by iop098321qwe** [BlossomMods/BlossomBack#5]
I am trying to get the /back command to work for last death. I am unfamiliar with how to change the config file. I have tried a few things but have resorted to giving up on getting that function to work.

Is it possible to get better instructions for exactly how to enable this feature?

**Comment by CodedSakura** [comment]
i agree that the way i've made the last-death command is not very user-friendly, but i'll have to think about how to change it to be easier to use

it basically only requires that you give the players the `blossom.last-death` permission (using LuckPerms or similar)

the config can remain the default one, as it will by default use the same config as the /back command

**Comment by iop098321qwe** [comment]
I have since installed LuckPerms actually. I have no idea how to use it and it is quite the learning curve, but I did get that one specific command that I wanted to work working.

It is probably worth it in the future for some other folks so they don't have to figure it out or have to run LuckPerms on a dedicated server for just a couple friends (like myself). I appreciate the timeliness on the response. I have no problem taking the time to learn it as I am still experimenting with mods before I host it for my friends and their friends.

You do great work and your mods are my first choice for the functions that you provide. I checked out the BlossomMods roadmap project and I am looking forward to this developing even further. I will have my welcome message in my server set to promote your mods (with a link of course). :)
